src/main/java/com/getify/minify/Minify.java

```diff
@@ -0,0 +1,103 @@
+//
+//  Minify.java
+//  jsonminifier - Ported from https://github.com/getify/JSON.minify
+//
+//  Created by Bernhard Gass on 8/01/13.
+//  Copyright © 2013 Bernhard Gass. All rights reserved.
+
+package com.getify.minify;
```

Comment:
@pradyunsg that can be changed, but if we want to use `com.json_minify` then we really should(/must) obtain the domain `json_minify.com`. The reason for this is that java package naming convention is to use the reversed internet domain as the package name. The low level domain `minify` for this project would correspond to `minify.getify.com`, while that domain might not actually exist, it's ok, because we control the second level domain `getify.org`.

**TL;DR:** Java convention for package naming is to use the reversed domain of the organization, followed by the project name.